bucket/cht.json

```
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/tpanj/cht.exe",
+    "description": "A command line client for https://cht.sh, working examples about various commands, APIs, ... Unified access to the best community driven documentation repositories of the world.",
+    "license": "GPL-3.0-or-later",
+    "version": "0.6",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/tpanj/cht.exe/archive/v0.6.zip#/bin/cht.exe",
```

That's not how it work.

`#/` is for renaming, not specifying what to extract.

```suggestion
            "url": "https://github.com/tpanj/cht.exe/archive/v0.6.zip",
```